src/index.ts

```diff
@@ -6,7 +6,9 @@ function makeReader(filename: string) {
 
 	return function(): Record<string, any>[] {
 		if (json.length === 0) {
-			json = JSON.parse(fs.readFileSync(resolve('.', filename + '.json'), 'utf-8'));
+			let path = resolve(__dirname, '..', filename + '.json');
+			console.log('path is ' + path);
```

Comment:
We should probably use a logging tool here, so consumers of the API don't have messages they don't want.

Reply:
the package `loglevel` is really good for this, tbh

Reply:
yeah, this was just me being sloppy at the end of the day.  This and the other method were there just to confirm what i thought was happening in the other repo.